tests/measurements/test_shots.py

```diff
@@ -75,9 +75,12 @@ def test_deepcopy(self):
 
     def test_None(self):
         """Tests the constructor when shots is None."""
-        shots = Shots(None)
-        assert shots.shot_vector == ()
-        assert shots.total_shots is None
+        shots1 = Shots(None)
+        shots2 = Shots()  # this also defaults to None
+        assert shots1.shot_vector == ()
+        assert shots2.shot_vector == ()
+        assert shots1.total_shots is None
+        assert shots2.total_shots is None
```

Comment:
Why this addition is needed?

Reply:
To test that the default `Shots()` is the same as `Shots(None)` since the default value of the first argument is `None`. Mostly for code coverage to pass 😆